app/models/user_list_upload/user_row.rb

```diff
@@ -44,6 +44,15 @@ def user
     end
   end
 
+  def user_for_display
+    # Skip uniqueness validations for display purposes to avoid N+1 queries
+    user.tap { |u| u.skip_uniqueness_validations = true }
+  end
+
+  def user_errors_for_display
```

Comment:
Du coup ce serait plus logique d'appeler la méthode
```suggestion
  def user_for_display_errors
```